package.json

```
@@ -12,8 +12,8 @@
     "test:watch": "npm test -- --watch",
     "build:watch": "cross-env WEBPACK_ENV=serve webpack serve --mode development --config ./webpack.config.dev.js",
     "test:coverage": "jest --coverage",
-    "cypress:cli": "npx cypress run-ct",
-    "cypress:ui": "npx cypress open-ct"
+    "cypress:cli": "node wrapCypress.ts cypress  run --component",
```

```suggestion
    "cypress:cli": "cypress run --component --browser=chrome",
```
We don't need to use the wrapCypress.ts